public/js/UX.js

```diff
@@ -116,7 +116,8 @@ function showProperties(obj) {
 
         $('#moduleProperty-inner').append("<div id='moduleProperty-header'>" + obj.objectType + "</div>");
         // $('#moduleProperty').append("<input type='range' name='points' min='1' max='32' value="+obj.bitWidth+">");
-        if (!obj.fixedBitWidth)
+
+        if (!obj.fixedBitWidth && obj.bitWidth)
```

Comment:
```suggestion
        if (!obj.fixedBitWidth && obj.bitWidth)
```
All objects have bitWidth so this is not exactly required.